.github/workflows/spellcheck.yml

```
@@ -0,0 +1,26 @@
+name: Spell checking
+on:
+  pull_request_target:
+  workflow_dispatch:
+
+jobs:
+  spelling:
+    name: Spell checking
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v3
+      - uses: check-spelling/check-spelling@v0.0.20-alpha7
+        id: spelling
+        with:
+          experimental_path: docs/whitepaper
```

@hleonardtek actually Anik should be the one commenting the PR

```suggestion
          GITHUB_TOKEN: ${{ secrets.OKP4_TOKEN }}
```